scipy/optimize/_shgo.py

```diff
@@ -87,14 +87,14 @@ def shgo(func, bounds, args=(), constraints=None, n=100, iters=1, callback=None,
         Extra keyword arguments to be passed to the minimizer
         ``scipy.optimize.minimize`` Some important options could be:
 
-            * method : str
-                The minimization method (e.g. ``SLSQP``).
-            * args : tuple
-                Extra arguments passed to the objective function (``func``) and
-                its derivatives (Jacobian, Hessian).
-            * options : dict, optional
-                Note that by default the tolerance is specified as
-                ``{ftol: 1e-12}``
+        method : str
+            The minimization method (e.g. ``SLSQP``).
+        args : tuple
+            Extra arguments passed to the objective function (``func``) and
+            its derivatives (Jacobian, Hessian).
+        options : dict, optional
+            Note that by default the tolerance is specified as
+            ``{ftol: 1e-12}``
```

Comment:
Line 101 should have `scipy.optimize.minimize` in backticks

Reply:
I will fix this one, but we could spend days fixing inconsistent use of backticks! We really need to decide what deserves double backticks and what should have just singles. E.g. [NumPy guidelines](https://numpy.org/doc/stable/docs/howto_document.html) say "Enclose variables in double backticks", but where do we draw the line between a "variable" and code? Why is it appropriate for the parameter names to be bold in the parameter list, italicized when we refer to them in the descriptions, and teletype when we actually type them in the IDE? If I had my way, all inline code would be teletype (bold/italicized as needed for emphasis) but without the gray box around, just as Matlab does it

![image](https://user-images.githubusercontent.com/6570539/99914007-a55a7580-2caf-11eb-8baf-136a88e01229.png)

But this is not the place : )

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/6570539/102744709-49ffbf80-430f-11eb-9188-8dd409012f2b.png)

https://23582-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.optimize.shgo.html#scipy.optimize.shgo